<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../icons.qrc">
    <normaloff>:/icons/collection-triangular-iteration.png</normaloff>:/icons/collection-triangular-iteration.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="main_label"/>
   </item>
   <item>
    <widget class="QLabel" name="graph_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../preview.qrc">:/icons/preview/PLAP.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="point_A_label">
       <property name="text">
        <string>A Point:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="point_A"/>
     </item>
     <item>
      <widget class="QLabel" name="point_B_label">
       <property name="text">
        <string>B Point:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="point_B"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../preview.qrc"/>
  <include location="../../../../icons.qrc"/>
  <include location="../../../../../../桌面/Pyslvs-PyQt5/preview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
